cmd/podman/run.go

```diff
@@ -72,12 +72,23 @@ func runCmd(c *cli.Context) error {
 	options = append(options, libpod.WithUser(createConfig.User))
 	options = append(options, libpod.WithShmDir(createConfig.ShmDir))
 	options = append(options, libpod.WithShmSize(createConfig.Resources.ShmSize))
+
+	// Default used if not overridden on command line
+
+	if createConfig.CgroupParent != "" {
+		options = append(options, libpod.WithCgroupParent(createConfig.CgroupParent))
+	}
+
 	ctr, err := runtime.NewContainer(runtimeSpec, options...)
 	if err != nil {
 		return err
 	}
 
 	logrus.Debug("new container created ", ctr.ID())
+
+	p, _ := ctr.CGroupPath()("")
```

Comment:
Is this just debugging code?  Or do you feel that we should always log the cgrouppath?

Reply:
I only feel strongly that we need this information for debugging should a container end up in an unexpected cgroup.

I can change the level if you feel this information is needed for all starts

Reply:
Debug is a good place for this, I think. Won't show up in the default log level, but there if we request debug logging

Reply:
I am thinking about this being called `p, _ := ctr.CGroupPath()("")` every time, and if the loglevel is not debug, it does nothing.  But if this does not take any real CPU I am fine with leaving it.